app/src/main/java/com/alexbar/layoutbasic/utils/MoviesData.kt

```diff
@@ -0,0 +1,63 @@
+package com.alexbar.layoutbasic.utils
+
+import android.content.Context
+import com.google.gson.Gson
+import com.google.gson.annotations.SerializedName
+import com.google.gson.reflect.TypeToken
+import java.io.IOException
+
```

Comment:
regarding this file 
In many big companies, you won't find everything lumped together in one file. Different things like data classes, functions, and so on are usually kept separate. So, I'll share some comments about this file with you.